schema_registry/client/schema.py

```diff
@@ -69,6 +69,7 @@ class AvroSchema(BaseSchema):
     def __init__(self, *args: typing.Any, **kwargs: typing.Any) -> None:
         self._expanded_schema: typing.Optional[typing.Dict] = None
         self._flat_schema: typing.Optional[typing.Dict] = None
+        self._ignore_default_error: typing.Optional[bool] = False
```

Comment:
You are missing the new property `ignore_default_error`in the `__init__`. `def __init__(self, *args: typing.Any, ignore_default_erro: bool = False, **kwargs: typing.Any) -> None: ...`

Probably we should pass all the `fastavro` properties here, but we can do it in a different PR. It would be nice to have some tests as well to check this change.